File: diff_calc.py
with open("Bloeddruk_en_muziek/data.csv") as file:
    file = file.readlines()
for line in file:
    if "id" in line:
        result = line
    else:
        boven_diff = int(line.split(',')[4]) - int(line.split(',')[2])
        onder_diff = int(line.split(',')[5]) - int(line.split(',')[3])
        bpm_diff = int(line.split(',')[7]) - int(line.split(',')[6])
        line = line.replace('\n','')
        result += f"{line}{boven_diff},{onder_diff},{bpm_diff},\n"
print(result)

File: Verschil_berekening/diff_calc.r
file <- readLines("Bloeddruk_en_muziek/diff_calc/diff_calc_input.csv")
result <- ""
for (line in file) {
  if ("id" %in% line) {
    result <- line
  } else {
    split_line <- strsplit(line, ",")[[1]]
    boven_diff <- as.integer(split_line[5]) - as.integer(split_line[3])
    onder_diff <- as.integer(split_line[6]) - as.integer(split_line[4])
    bpm_diff <- as.integer(split_line[8]) - as.integer(split_line[7])
    line <- gsub("\n", "", line)
    result <- paste0(result, line, boven_diff, ",", onder_diff, ",", bpm_diff, "\n")
  }
}
cat(result)
